server/container/cli-integration/src/test/java/org/apache/james/cli/DataCommandsIntegrationTest.java

```diff
@@ -146,6 +148,8 @@ public void addAddressMappingShouldWork() throws Exception {
 
     @Test
     public void listMappingsShouldWork() throws Exception {
+        dataProbe.addDomain(DOMAIN);
```

Comment:
can you add some test to demonstrate what happens with the cli if the domain is missing?